angular-ngrx-scss/src/app/state/user/user.state.ts

```diff
@@ -10,6 +12,8 @@ export interface UserState {
   name: string;
   twitter_username: string;
   username: string;
+  topRepos?: UserReposState[];
+  gists?: UserGistsState[];
```

Comment:
suggestion: I think these should be attached to home page specific state rather than being a part of the global user state; however I know we discussed possibly refactoring some of the state management in this showcase, so we can elect to fix that later if that makes more sense.

Reply:
I would like to make these adjustments as part of a larger overhaul to how the store is set up, as we discussed. I think it's worth leaving them for now and making a new ticket to correct the state structures all over. Will keep those changes easier to review, as well, I think.

Reply:
Ok! Makes sense to me.